Site/ASAB Maestro/Descriptors/burrow.yaml

```diff
@@ -0,0 +1,43 @@
+
+define:
+  type: rc/descriptor
+  name: Burrow
+
+descriptor:
+  image: docker.teskalabs.com/burrow-maestro
```

Comment:
`docker.teskalabs.com/burrow-maestro` .... this is our image?

Reply:
Funny thing nobody ever wondered if `docker.teskalabs.com/burrow` was our image, and it has been there for years... But yes, this one is a burrow that works with maestro without mangling the remote-control. I can do a proper version release after discussing which of the (I guess we identified 4 options how to do this integration) options is most suited to our needs.

Reply:
I'm ok with our image for Burrow - if we do an update to the most recent version time to time.

Also, it would be great to unify version so that the release is not specific to maestro.

Reply:
Sure, I can do version releases that are in line with our naming and do an update from time to time (the burrow is pretty alive, it seems linkedin uses it a lot, or cares about it). I just added a support for "our way" of providing a zookeeper and kafka configuration.